ckks/encryptor.go

```diff
@@ -3,394 +3,54 @@ package ckks
 import (
 	"github.com/ldsec/lattigo/v2/ring"
 	"github.com/ldsec/lattigo/v2/rlwe"
-	"github.com/ldsec/lattigo/v2/utils"
 )
 
-// Encryptor in an interface for encryptors
-//
-// encrypt with pk : ciphertext = [pk[0]*u + m + e_0, pk[1]*u + e_1]
-// encrypt with sk : ciphertext = [-a*sk + m + e, a]
-type Encryptor interface {
```

Comment:
It is probably better to keep an interface at the scheme's level.

Reply:
fixed